docs/content/users/configuration/experimental.md

```diff
@@ -6,4 +6,6 @@ DDEV v1.22+ supports Python-based projects, including those built with Django 4
 
 `ddev config --project-type=django4` will by default a project to use the `nginx-gunicorn` `webserver_type` and the `postgres` database type.
 
+To get started with either one of these Python projects, checkout the [Django4 Quickstart](../quickstart/#django-4-experimental) or the [Flask Quickstart](../quickstart/#pythonflask-experimental).
```

Comment:
This needs a space to be consistent with the page it links to and [Django’s own formatting](https://docs.djangoproject.com/en/4.1/releases/4.0/): 

```suggestion
To get started with either one of these Python projects, checkout the [Django 4 Quickstart](../quickstart/#django-4-experimental) or the [Flask Quickstart](../quickstart/#pythonflask-experimental).
```